src/app/games/tafl/TaflMove.ts

```diff
@@ -0,0 +1,73 @@
+import { Coord } from 'src/app/jscaip/Coord';
+import { MoveCoordToCoord } from 'src/app/jscaip/MoveCoordToCoord';
+import { Direction } from 'src/app/jscaip/Direction';
+import { NumberEncoder } from 'src/app/jscaip/Encoder';
+
+export class TaflEncoder<M extends TaflMove> extends NumberEncoder<M> {
+
+    constructor(public readonly MAX: number,
+                public readonly instanceProvider: (start: Coord, end: Coord) => M) {
```

Comment:
shouldn't MAX & instanceProvider be private?

Reply:
instanceProvider is just a reminder that Tafl Children Component will need to have it, and that here is its rightfull place, it's a reference to a [must be] public method anyway.

Reply:
Done for both, btw.